src/main/scala/slamdata/SbtSlamData.scala

```diff
@@ -282,5 +357,20 @@ object SbtSlamData extends AutoPlugin {
         currentVersion + "-" + "git rev-parse HEAD".!!.substring(0, 7)
       else
         currentVersion
-    })
+    },
+
+    Test / test := (Def.taskDyn {
+      val t: Unit = (Test / test).value
+      if (name.value == "root" && isTravisBuild.value) {
```

Comment:
Worth noting that `"root"` is just a convention, not actually enforced by sbt, though we never violate it. This is fine.

Reply:
I know, I just don't know how else to do it.